src/bidder_submitter.ts

```diff
@@ -17,13 +17,18 @@ export enum BidderSubmissionType {
   UNWIND = 'unwind',
 }
 
-export interface AuctionBid {
+export interface BaseBidderSubmission {
+  type: BidderSubmissionType;
+  poolConfig: PoolConfig;
```

Comment:
with the config changes, this can likely be updated to just include `poolAddress`